src/block/posts/util.js

```diff
@@ -199,7 +199,19 @@ export const generateRenderPostItem = attributes => {
 
 		// Trim the excerpt.
 		let excerptString = postExcerptStackable.split( ' ' )
-		if ( excerptString.length > ( excerptLength || 55 ) ) {
+
+		//Checks if there are CJK characters present
+		const regexCJK = /[\u3040-\u30ff\u3400-\u4dbf\u4e00-\u9fff\uf900-\ufaff\uff66-\uff9f]/
+		if ( postExcerptStackable.match( regexCJK ) ) {
+			// Extract the string inbetween the tags and newline.
+			const tempExcerptString = postExcerptStackable.match( '<p>(.*)<\/p>\\n' )
```

Comment:
The string that I obtained when I would use the `postExcerptStackable` would contain the `<p>`  which is why I made it get the string between those tags and the reason why I made the function `substing(3,excerptLength+3)` start at 3 before and then also added a +3. Same reason for the `$untrimmed_excerpt` in the `php` file.

Reply:
The reason why I also made an `if (excerptLength === 1 ) ...` back then was because it would look like this in the backend if the excerpt length is 1:
![image](https://user-images.githubusercontent.com/49634311/167697638-33c63ef0-358d-452d-83e1-e1e5a948f8ca.png)